.github/workflows/reusable-crn-algolia-sync.yml

```diff
@@ -100,120 +100,15 @@ jobs:
           ALGOLIA_API_KEY: ${{ steps.algolia-keys.outputs.algolia-api-key }}
           ALGOLIA_APP_ID: ${{ steps.algolia-keys.outputs.algolia-app-id }}
           ALGOLIA_INDEX_TEMP: '${{ steps.setup.outputs.crn-algolia-index }}_${{ github.run_id }}_temp'
-          ENTITY_TYPE: ${{ (inputs.entity == 'users' && 'user') || (inputs.entity == 'research-outputs' && 'research-output') || (inputs.entity == 'external-authors' && 'external-author') || (inputs.entity == 'events' && 'event') }}
-      - name: Import Research Outputs (ROs | ALL)
-        if: ${{ inputs.entity == 'research-outputs' || inputs.entity == 'all'}}
+          ENTITY_TYPE: ${{ (inputs.entity == 'users' && 'user') || (inputs.entity == 'research-outputs' && 'research-output') || (inputs.entity == 'external-authors' && 'external-author') || (inputs.entity == 'events' && 'event') || (inputs.entity == 'interest-groups' && 'interest-group') || (inputs.entity == 'news' && 'news') || (inputs.entity == 'teams' && 'team') || (inputs.entity == 'tutorials' && 'tutorial') || (inputs.entity == 'working-groups' && 'working-group') }}
```

Comment:
since you are refactoring I thought maybe there is a better way to map these values? like a switch or a Map object we could reference or something? it's not necessary though